### PR TITLE
99 Create Settings Page

### LINKED ISSUE #99: Create Settings Page
# Goal

Create the GUI settings page to let the user set variables within the pipeline.

# Plan

1. Let the user set the threshold confidence value.
2. Let the user select the frame rate.
3. Let the user determine the ideal posture.

### PR DESCRIPTION
**Pull request by Andrew-Ritchie**
# Details

- Let the user change the posture sensitivity through a slider input.
- Let the user adjust the video frame rate.
- Let the user set the current pose as the ideal posture.

# Screenshots
Screenshot 2021-03-26 at 15 24 28

# Checklist

## Code
- [ ] I have added some tests
- [x] I have run the tests locally to ensure they all pass by running `.scripts/build.sh -enable-testing`

## Documentation
- [ ] I have updated documentation appropriately (GitHub Pages)
- [ ] I have tested the updated GitHub Pages locally to ensure the changes to documentation render correctly (see this [Guidance](https://ese-peasy.github.io/PosturePerfection/guidance.html) page)

Closes #99

### CODE REVIEW

File: src/pipeline.cpp [hunk 1]
```diff
@@ -25,6 +25,8 @@
 #define MODEL_INPUT_X 224
 #define MODEL_INPUT_Y 224
 
+using namespace std;
```
Comment (Ashwin-MJ) [inline src/pipeline.cpp]: Could we take this out?

**Review by Ashwin-MJ — COMMENTED** [review]
One comment but otherwise looks great!

File: src/pipeline.cpp (hunk 1 above)
Comment (Andrew-Ritchie) [reply src/pipeline.cpp]: no problem, that was used for testing but has been removed.

**Review by miklasr — APPROVED** [review]
This looks good to me, with one minor point. I won't block merging for this, though, as we can address it for the next milestone.

The buttons and sliders on the settings page appear black/grey when running on my laptop (same issue as Conor had with the drop-down menu at some point). Could we make these consistent with other buttons etc. in the GUI?

I'm happy for this to be done in another issue for the next milestone. If you agree, can you add it to the list of things to do for #89?

**Comment by Andrew-Ritchie** [comment]
Yes i agree! Thanks @miklasr. :)

**Comment by C-Begley** [comment]
> This looks good to me, with one minor point. I won't block merging for this, though, as we can address it for the next milestone.
> 
> The buttons and sliders on the settings page appear black/grey when running on my laptop (same issue as Conor had with the drop-down menu at some point). Could we make these consistent with other buttons etc. in the GUI?
> 
> I'm happy for this to be done in another issue for the next milestone. If you agree, can you add it to the list of things to do for #89?

I have the same issue. If it's white on the Pi, it should be fine, as that's the target device. I'm happy to approve

**Review by C-Begley — APPROVED** [review]

**Review by C-Begley — APPROVED** [review]

**Pull request merged**